README.md

```diff
@@ -0,0 +1,140 @@
+# FMS HF Tuning
+
+This repo aims to create basic tuning scripts with support for specific models, assumes the use of Hugging Face `SFTTrainer` and PyTorch FSDP only. Our approach to tuning is:
+1. Models are loaded from Hugging Face `transformers` or the `foundation-model-stack` -- models are either optimized to use `Flash Attention v2` directly or through `SDPA`
```

Comment:
for `foundation-model-stack` might be better to either use the pypi name (ibm-fms) or link to the github repo